tests/library/trace-viewer.spec.ts

```diff
@@ -102,6 +111,67 @@ test('should open trace viewer on specific host', async ({ showTraceViewer }, te
   await expect(traceViewer.page).toHaveURL(/127.0.0.1/);
 });
 
+test('should show groups as tree in trace viewer', async ({ showTraceViewer }) => {
```

Comment:
ok. 
I thought "why adding runtime by creating a new trace, if the common trace just does it".
But that is fine.
I had already the source location tested, but i will also create a function so that the group will have a call stack which is needed to show the list like `should show action source` is testing.

Reply:
small correction:
The `.stack-trace-list-view` is only visible if there is more than one location in the `stackFrames`.
Groups and also test.step do just have their own location in the `stackFrames`.

So that stack-trace is either never visible for groups or we change the behaviour in both. 

For now i continue with checking the code line in the `Source` tab in the editor title and the line.

Reply:
Test is done.